### PR TITLE
Tunnel electron logs to the spot-client logger

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+/**
+ * Log levels supported by the jitsi logger.
+ */
+const levels = [ 'trace', 'debug', 'info', 'log', 'warn', 'error' ];
+
+const LOG_CACHE_SIZE = 1000;
+
+/**
+ * A class
```
Comment (virtuacoplenny) [inline spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js]: Yes

File: spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ */
+class SpotClientLogTransport {
+    /**
+     * Creates a new instance of {@code LoggingService} and registers itself
+     * to receive logs.
+     */
+    constructor() {
+        for (const level of levels) {
+            this[level] = this.log.bind(this);
```
Comment (virtuacoplenny) [inline spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js]: Is there a need to differentiate the different levels of logging?

File: spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+     * The log message executed for every logging level.
+     *
+     * @param {...*} args - Log method arguments as defined by jitsi-logger.
+     * @returns {void}
+     */
+    log(...args) {
+        const logString = args.join(' ');
+
+        if (clientController.canSendClientMessage()) {
+            clientController.sendClientMessage('spot-electron-logs', logString);
```
Comment (virtuacoplenny) [inline spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js]: Do you think logging the context object will be needed soon?

Comment (paweldomas) [reply spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js]: yes

File: spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js (hunk 2 above)
Comment (paweldomas) [reply spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js]: yes

Comment (virtuacoplenny) [reply spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js]: I was not being clear. I meant right now only the level "log" gets logged. Do you also need to tell the client right now to send error or warn or other levels?

Comment (paweldomas) [reply spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js]: I think this should support all levels, yes

File: spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js (hunk 1 above)
Comment (virtuacoplenny) [reply spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js]: Let's never change this.

**Pull request merged**